### PR TITLE
Code changes necessary for SCM versioning

### PR DESCRIPTION
**Pull request by jzuhone**
This PR incorporates the code changes required for version numbers to be handled by git. This will allow version numbers to be set by git tags, and in between releases development version numbers will be given.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+.git_archival.txt  export-subst
```
Comment (jeanconn) [inline .gitattributes]: @javierggt does this need newline or fine without it?

Comment (javierggt) [reply .gitattributes]: I don't know for sure, but I believe it is not necessary

Comment (jeanconn) [reply .gitattributes]: OK.  @jzuhone I don't have a quick way to test that so I think it is fine as well; if we end up with any weird things when we build packages we can fix later.

File: acis_thermal_check/__init__.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
-__version__ = "2.9.0"
```
Comment (jeanconn) [inline acis_thermal_check/__init__.py]: @jzuhone As an aside, I note that if ACIS has any code that is importing the modules to check version, I expect with with setuptools-scm the 'in-the-code' versions will now have your 'v' prefix if you continue to make github releases with it (like 'v2.9.0').

Comment (jzuhone) [reply acis_thermal_check/__init__.py]: We plan on dropping the "v" prefix from now on. Thanks for the heads-up, @jeanconn.

**Pull request merged**